CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # stackstorm cookbook CHANGELOG
 
 ## In Development
+## 0.7.1
```

Comment:
Let's please place this fix under the `v0.7.0` changelog as I didn't release new version after the most recent PR #74

Reply:
Sure thing!